visualization.py

```
@@ -0,0 +1,57 @@
+import networkx as nx
+from pyvis.network import Network
+import matplotlib
+matplotlib.use('Agg')  # Set the backend to Agg before importing pyplot
+import matplotlib.pyplot as plt
+from hashing import hash_query, decode_answer
```

Ruff (0.8.2)

`hashing.hash_query` imported but unused

Remove unused import: `hashing.hash_query`

(F401)